composer.json

```diff
@@ -1,14 +1,18 @@
 {
     "name": "guzzlehttp/psr7",
     "type": "library",
-    "description": "PSR-7 message implementation",
-    "keywords": ["message", "stream", "http", "uri"],
+    "description": "PSR-7 message implementation that also provides common utility methods",
+    "keywords": ["request", "response", "message", "stream", "http", "uri", "url"],
     "license": "MIT",
     "authors": [
         {
             "name": "Michael Dowling",
             "email": "mtdowling@gmail.com",
             "homepage": "https://github.com/mtdowling"
+        },
+        {
+            "name": "Tobias Schultze",
```

Comment:
I hope that is ok considering the amount of time I've invested in improving this library :)